example-deployment/docker/minimal.dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ros:foxy-ros-base-focal
+
+RUN apt-get update && apt-get install -y curl && \
+  curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update && apt-get install -y nodejs python3-pip
+
+RUN pip3 install pipenv
+
+COPY . /root/rmf-web
+SHELL ["bash", "-c"]
+
+RUN npm config set unsafe-perm && cd /root/rmf-web && \
+  CI=1 npm install -g lerna@4 && lerna bootstrap --scope=minimal
+
+RUN cd /root/rmf-web/packages/minimal && \
+  PUBLIC_URL='/minimal' \
+  REACT_APP_REPORTING_SERVER='https://example.com/logserver/api/v1' \
```

Comment:
is there a reason for REPORTING_SERVER to be in this package?

Reply:
addressed!